tutorials/TimePicker.md

```diff
@@ -0,0 +1,65 @@
+Here is an overview about how to use the TimePicker page object:
+
+    const PageTimePicker = require('react-rainbow-components/components//TimePicker/pageObject');
+    const { ESCAPE_KEY } = require('../../constants');
```

Comment:
look at other tutorials as we define the key code directly since this is documentation for user and not make sense:
```require('../../constants');```